pkg/asttransform/baseschema.go

```diff
@@ -119,7 +119,7 @@ func addTypeField(definition *ast.Document) (ref int) {
 }
 
 func findQueryNode(definition *ast.Document) (queryNode ast.Node, ok bool) {
-	queryNode, ok = definition.Index.Nodes[xxhash.Sum64String("Query")]
+	queryNode, ok = definition.Index.Nodes[xxhash.Sum64(definition.Index.QueryTypeName)]
```

Comment:
I don't know but isn't there a `Mutation`, `Subscription` etc. equivalent of this ?